app/internal/security/ouath2.py

```diff
@@ -69,10 +69,8 @@ async def check_jwt_token(
     try:
         jwt_payload = jwt.decode(
             token, JWT_KEY, algorithms=JWT_ALGORITHM)
-        if not manager:
-            return True
-        if jwt_payload.get("is_manager"):
-            return True
+        if not manager or jwt_payload.get("is_manager"):
```

Comment:
This line is problematic. Might be a simple naming issues - but currently it contains a logic contradiction.